### PR TITLE
Add changelog for NewPipe 0.22.0 (983)

### PR DESCRIPTION
**Pull request by TobiGr**
Adds the changelog for the upcoming 0.22.0 version.
This enables the translators on Weblate to work on localizations before the new version is published

### CODE REVIEW

File: fastlane/metadata/android/en-US/changelogs/983.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+New fast forward / rewind behaviour
```
Comment (opusforlife2) [inline fastlane/metadata/android/en-US/changelogs/983.txt:1]:
```suggestion
New fast forward / rewind behaviour and UI
```

Comment (mhmdanas) [reply fastlane/metadata/android/en-US/changelogs/983.txt:1]: @opusforlife2 somehow it looks better the other way around:
```suggestion
New fast forward / rewind UI and behavior
```

File: fastlane/metadata/android/en-US/changelogs/983.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Made settings searchable
+Highlight pinned comments as such
+Added FSFE PeerTube instance to URL opener
+Improved error reporting and new error notification
+Fixed first item always played in the play queue when reloading play queue manager
```
Comment (TiA4f8R) [inline fastlane/metadata/android/en-US/changelogs/983.txt:6]: Probably most of users don't know what is the play queue manager and where/when/how it is used so I'd rename this entry to:

```suggestion
Fixed first item always played in the play queue when switching quality or player type
```

File: fastlane/metadata/android/en-US/changelogs/983.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+New fast forward / rewind behaviour
+Made settings searchable
+Highlight pinned comments as such
+Added FSFE PeerTube instance to URL opener
+Improved error reporting and new error notification
```
Comment (TiA4f8R) [inline fastlane/metadata/android/en-US/changelogs/983.txt:5]: The error notification has been added and not improved, so I think we should reflect (properly) this in the changelog:

```suggestion
Improved error reporting and add new error notification
```

**Review by mhmdanas — CHANGES_REQUESTED** [review]
I think the whole changelog is a little inconsistent. Sometimes there's no verb at all ("New fast forward / rewind behaviour"), other times an imperative verb ("Highlight pinned comments as such"), and others a past verb ("Added FSFE PeerTube instance to URL opener"). We should choose one style and stick to it.

**Comment by TobiGr** [comment]
@opusforlife2 Looks good to me!

**Comment by mhmdanas** [comment]
I prefer the imperative, as @Stypox suggests.

**Comment by TobiGr** [comment]
@mhmdanas I don't have time to work on this, feel free to push a commit or add your version as a comment.

**Comment by TobiGr** [comment]
I reabsed, we should squash + merge this

**Review by mhmdanas — APPROVED** [review]
LGTM

**Pull request merged**